internal/pkg/service/stream/storage/node/coordinator/fileimport/operator.go

```diff
@@ -91,6 +104,11 @@ func Start(d dependencies, config targetConfig.OperatorConfig) error {
 		statistics: d.StatisticsL1Cache(),
 		locks:      d.DistributedLockProvider(),
 		plugins:    d.Plugins(),
+		meters: &meters{
+			duration:     d.Telemetry().Meter().FloatHistogram("keboola.go.stream.operation.duration", "Duration of operator processing.", "ms"),
+			compressed:   d.Telemetry().Meter().IntHistogram("keboola.go.stream.operation.bytes.compressed", "Compressed bytes processed by operator.", "B"),
+			uncompressed: d.Telemetry().Meter().IntHistogram("keboola.go.stream.operation.bytes.uncompressed", "Uncompressed bytes processed by operator.", "B"),
+		},
```

Comment:
The metrics have to include operation name, if they will be defined in the each operator.

Here, you should also set buckets for the histogram, these operations are longer.

WithExplicitBucketBoundaries(.05,.1, .15, .2,.3,.4,.5,.75,1, 2.5, 5, 7.5, 10,20,30,60,90,120,240,300,600),